packages/sitecore-jss-react/src/components/BYOCComponent.test.tsx

```diff
@@ -13,10 +13,9 @@ describe('BYOCComponent', () => {
       },
     };
     const wrapper = mount(<BYOCComponent {...mockProps} />);
-    const externalComponent = wrapper.find('Se');
+    const externalComponent = wrapper.find('feaas-external');
     expect(externalComponent).to.have.lengthOf(1);
-    expect(externalComponent.prop('componentName')).to.equal('ExternalComponent');
-    expect(externalComponent.prop('prop1')).to.equal('value1');
+    expect(externalComponent.prop('data-external-id')).to.equal('ExternalComponent');
```

Comment:
We should be testing that props are passed down here as well

Reply:
I didn't find a way how to verify that. It's a good point, but in a latest version of 'feaas' sdk they made updates and only data-external-id is possible to validate, _ComponentProps_ are not present when component is not rendered by feaas library

Reply:
And for some reason there is no way to access _FEAAS.ExternalComponent_ props, but it's rendered

Reply:
@Inviz is there a way to verify props have been passed to External Component (in unit tests in this case)?

Reply:
All props should be observable on feaas-external element, if the component was found.

```
      <feaas-external
        {...objectKeysToKebabCase(parsed)}
        data-external-id={componentName}
        suppressHydrationWarning
        hydrate={'false'}
      >
        {Component == null ? null : <Component {...parsed} />}
      </feaas-external>
```
https://bitbucket.org/stylelabsdev/feaas-components/src/a2ddc1427c413e939b3cd4a64120db1bfe73e8be/clientside/src/ui/react.tsx#lines-128:153

they are to be seen in kebab case though.

Reply:
btw i noticed this mention accidently on email, not checking github often. please hitme up directly in case i dont see something.

Reply:
@Inviz You are right, using unit tests we have a case when the component is not registered, so I couldn't verify props...
I decided to register component in the unit test and now it works fine, I could verify all the parts

cc @art-alexeyenko

Reply:
see here: https://github.com/Sitecore/jss/pull/1605/files#diff-5bae97692bcb1065c314a8f2988f8842986cdbc8eb90ac9a3f979156091f1149R17-R24